src/vtbackend/ViCommands.cpp

```diff
@@ -773,8 +782,13 @@ CellLocation ViCommands::globalCharDown(CellLocation location, char ch, unsigned
     return result;
 }
 
-CellLocation ViCommands::translateToCellLocation(ViMotion motion, unsigned count) const noexcept
+CellLocation ViCommands::translateToCellLocation(ViMotion motion, unsigned count) noexcept
```

Comment:
requiring both functions to be mutable sounds like a bad design choice. they were meant to *only* translate. But I do think I know why you where doing it, as these functions translate AND record cell location.

So either we factor out the recording of the current cell location, or (IMHO), I'd suggest to rename those function to something e.g. like `translateToCellLocationAndRecord`. What do you think?

Reply:
renamed to `translateToCellLocationAndRecord`